src/sql/optimizer/dynamic_partition_pruning.rs

```diff
@@ -536,7 +537,7 @@ fn read_table(
                 .project(projection.clone())
                 .ok();
             if let Some(row_iter) = row_iter_result {
-                rows.extend(row_iter);
+                rows.extend(row_iter.map(|r| r.expect("Parquet error encountered")));
```

Comment:
cc @sarahyurick just noting that with the changes pulled in by ADP 28, the row iterator here now gives `Result<Row, ParquetError>` instead of just the `Row` - I've gone ahead and made this minimal change to unblock compilation without changing much, not sure if there's a better way of handling things here

Reply:
Thanks! Looks fine to me.